src/components/tab-bar/tab.types.ts

```diff
@@ -0,0 +1,26 @@
+export interface Tab {
+    /**
+     * Name of the tab.
+     */
+    name: string;
```

Comment:
We might want to consider renaming this to `text` to be consistent with our other interfaces with similar functionality. Types like `ListItem`, `Chip` and `Option` all have the `text` property